meson.build

```diff
@@ -33,6 +33,7 @@ man_pages = get_option('man-pages')
 support_cgroups = get_option('support-cgroups')
 use_utmpx = get_option('use-utmpx')
 use_initgroups = get_option('use-initgroups')
+libselinux = dependency('libselinux', version : '>= 2.1.9', required : get_option('selinux'))
```

Comment:
There are two things:

1. For more clean meson.build, It's recommended to use `get_option()` and then saving it into a variable:
 ```meson
selinux = get_option('selinux')
 ```
2. See my comment about `## Dependencies` section in https://github.com/davmac314/dinit/pull/404#pullrequestreview-2397597547. I think it's better to be after `## Use -lrt`